src/ViewLayer/DisplayDashboard/DisplayDashboardUI/DisplayDashboardUI.cpp

```diff
@@ -21,7 +21,7 @@ DisplayDashboardUI::DisplayDashboardUI()
     //Font loading
     QApplication::setFont(fontLoader_->loadFont(Font::BURLINGAME));
     ui_->actualSpeedLabel->setFont(fontLoader_->loadFont(Font::LCD, 45, true));
-    ui_->stateOfChargeCapacityWidget->setFont(fontLoader_->loadFont(Font::LCD, 20, false));
+    ui_->stateOfChargeCapacityWidget->setFont(fontLoader_->loadFont(Font::BURLINGAME, 20, false));
```

Comment:
I meant that inside the `ui` file LCD was set in that stylesheet, you can undo this change :)